CONVENTIONS.md

```diff
@@ -116,12 +116,85 @@ Name screen coordinates `x` and `y`, rather than `left` and `top`.
 
 ## Javadocs
 
-Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs.
+Write sentences for class, method and fields javadocs, starting with an uppercase and ending with a period. Start method docs with verbs, like `Gets` or `Called`. Use HTML tags such as `<p>` if the docs have several paragraphs, as line wraps are converted to spaces in the generated documentation. Feel free to start a new line whenever you feel the current line is too long.
```

Comment:
Hmm, this is hard to do because indents may differ in different compilers

Reply:
I would say to start a new line for sentences rather than a hard character limit. Mappings don't need the same kind of readability as normal code, so they should at worst not have any wrapping at all. The middle ground is to have concise sentences and to wrap based on that instead.

Reply:
Hmm, currently my rule is wrap when the javadoc window runs out. Maybe we should specify that in the future (limit the field width in the javadoc window in enigma)

Reply:
Line widths preferably would be invisible to Enigma users, but that would require changes in Enigma.